### ISSUE TITLE
[BUG] Draft -> Post failure

### ISSUE THREAD

**Issue by theangryweasel** [CherpChat/IssueBoard#8]
I have a prompt that refuses to post despite multiple attempts to post it. I've worked on it several times over the past week, and saved and resaved it as a draft, but now that I want to post it, it won't work. I've checked to make sure its set to post, not save as draft multiple times. Is this a bug I'm unaware of, or something I'm doing wrong?

**To Reproduce**
1. Go to 'My Prompts'
2. Click on 'Edit'
3. Scroll down to 'Submit' and click
4. Be taken to the Directory, nothing has posted.
5. Return to 'My Prompts' to check timer, still "OK" meaning nothing has posted. 

**Expected behavior**
To post my prompt to the front page of the directory as it should.

**Screenshots**
N/A

**Desktop (please complete the following information):**
 - OS: Windows 10 Pro
 - Browser: Chrome
 - Version: 87.0.4280.141

**Smartphone (please complete the following information):**
 - Device: Samsung galaxy 10+
 - OS: Android 9.0
 - Browser: Chrome 
 - Version: 87.0.4280.141

**Comment by two-keys** [comment]
I'm going to reiterate some of Cherp's functionality, just to ensure that we're 100% on the same page and so I can trace where the breakpoint might be. Your wording in the in the 'expected behavior' field makes me believe this is a matter of mistaken terminology- sincere apologies if it isn't and I've misunderstood your bug. As far as I understand this: You have a prompt you hit Submit on, while having Draft selected, and are now trying to update its timestamp and therefore its placement in the directory.

There are two options you can select from when you go to edit a prompt.
`Posted - This request will be visible to everyone.`
`Draft - This request is visible only to you.`

There are the same options you have when you go to post a prompt for the first time.

If you're posting a prompt for the first time, selecting Draft 'posts' the prompt, but makes it only visible to you. In this sense, the prompt is logged to the database at the time you Submitted it as a 'Draft'- the timestamp is set to the submission time even if you later change this selection.
If you're editing an existing prompt, which seems to be the case here, selecting Draft ensures that the prompt is only visible to you, but doesn't modify its timestamp. Similarly, hitting 'Posted' when editing an existing prompt ensures that the prompt is now visible to anyone, but doesn't modify its timestamp.

The only way to modify a prompt's timestamp, then, is to go to My Prompts and hit 'bump' on the prompt(a button that should appear on a given prompt, to the right of the Edit link. See [this](https://user-images.githubusercontent.com/58573261/105394778-9d518000-5be3-11eb-880c-3e29bc32f79c.png)

.). Assuming that the prompt isn't a draft, indicated by a 'Draft' text at the top right, the prompt should have its timestamp updated and be 'bumped'.

I'm going to define 'submitted' as 'Hit submit on a new prompt with either the Draft or Posted option selected' from here on, to ensure we're on the same page. 
I would like to know:
1) Has it been five days since you've submitted this prompt? The bump button, which updates the timestamp and placement in the directory, won't appear unless it's been five days. This (unfortunately) means that saving a new prompt as a Draft and setting it to posted two days later... will leave it two days behind in the Directory, until the full five days pass and you can hit Bump.
2) If it has been five days, do you see either a Bump or a Delete button to the right of the Edit Link when you're seeing your prompt on the [My Prompts](https://cherp.chat/directory/mine) page?
3) If the Bump button exists, does clicking it properly update the timestamp?

TL;DR Hitting 'Bump' is the only way to make a prompt appear at the top of the directory, even if you originally posted it as a Draft. Setting a prompt to Posted only makes it visible- it doesn't bring it to the top of the Directory.
